<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScholarWidget</class>
 <widget class="QWidget" name="AddScholarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增奖学金</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SIMS_res.qrc">
    <normaloff>:/SIM_System_Pics/resources/icon_login_2.png</normaloff>:/SIM_System_Pics/resources/icon_login_2.png</iconset>
  </property>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>奖学金名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MoneyEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>奖学金金额</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="IntroEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>奖学金介绍</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="ConfirmButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="QuitButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="SIMS_res.qrc"/>
 </resources>
 <connections/>
</ui>
